lib/grids_page.dart

```diff
@@ -16,6 +16,12 @@ class GridsPage extends StatelessWidget {
 
   final List boardsList = boardTemplates.keys.toList();
 
+  int getCrossAxisCountByWidth(width) {
```

Comment:
Should this method be private in this library?
Avoid methods that starts with `get`: https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

Reply:
Fixed